include/ideep/operators/conv.hpp

```diff
@@ -51,7 +51,7 @@ struct convolution_forward_params {
   // TO-DO: Use a better name, i.e., num_threads
   int pd_use_threads;
   // Param for static quantization
-  std::shared_ptr<convolution_forward_quant_params> sq_param_ptr;
+  std::shared_ptr<convolution_forward_quant_params> sq_param_ptr = nullptr;
```

Comment:
`std::shared_ptr` is constructed as `nullptr` by default? Why did you see perf diff with this change?

Reply:
I thought it was initialized to `nullptr` by default. But if I explicitly assign it to `nullptr` here, perf numbers are slightly better.